testdata/runtests.sh

```diff
@@ -48,13 +48,47 @@ $EXE -c ../testutf3.cfg ../mixutf.crs mixutf3.out testutf3.log
 $EXE -c ../testutf4.cfg ../mixutf.crs mixutf4.out testutf4.log
 $EXE -c ../testutf5.cfg ../lolutf.crs lolutf.out lolutf.log
 
+# OutputFile as '-' Checks
+$EXE -c ../test1.cfg ../par1.crs - par1stdout.log > par1.out
+$EXE -c ../test2.cfg ../par1.crs - par2stdout.log > par2.out
+$EXE -c ../test3.cfg ../par1.crs - par3stdout.log > par3.out
+$EXE -c ../test1.cfg ../escape.crs - escapestdout.log > escape.out
+$EXE -c ../test4.cfg ../par1.crs - par4stdout.log > par4.out
+$EXE -c ../test5.cfg ../par1.crs - par5a.log 
+$EXE -c ../test5.cfg ../parh.crs - par5bstdout.log > par5b.out
+$EXE -c ../test5.cfg ../parhz.crs.gz - par5cstdout.log > par5c.out
+$EXE -c ../test6.cfg ../par1.crs - par6astdout.log > par6a.out
+$EXE -c ../test6.cfg ../par1.crs - par6bstdout.log > par6b.out
+$EXE -c ../test1.cfg -p ../scg.crs.p0+../scg.crs.p1.gz ../scg.crs - scgpstdout.log > scgp.out
+$EXE -c ../test1.cfg ../scg.crs+../scg1.crs - scg1stdout.log > scg1.out
+$EXE -c ../test1.cfg -l 0 scg - scg2stdout.log > scg2.out
+$EXE -c ../test1.cfg -x -l 0 -d 20100401030201 scg - scg5stdout.log > scg5.out
+
+$EXE -c ../testutf1.cfg ../mixutf.crs - testutf1stdout.log > mixutf1.out
+$EXE -c ../testutf5.cfg ../lolutf.crs - lolutfstdout.log > lolutf.out
+$EXE -c ../test1.cfg -w user_modify_flag=N ../scg.crs - scgw1stdout.log > scgw1.out
+
+# Test diff paramater options.
+$EXE -c ../test1.cfg -o id:area:shape ../par1.crs - par1o.log > par1o.out
+$EXE -c ../test1.cfg -a ../par1.crs - par1a.log > par1a.out
+$EXE -c ../test1.cfg -n ../par1.crs - par1n.log > par1n.out
+$EXE -c ../test1.cfg -h ../par1.crs - par1h.log
+
+# Checks no output to stdout and gets first line of stderr.
+$EXE -c ../test1.cfg ../par1.crs -+ par1f1.out 2>&1 > stdout | head -n1 > par1f1.log
+if [ ! -s stdout ]; then rm stdout; fi
+$EXE -c ../test1.cfg ../par1.crs par1f2.out - 2>&1 > stdout2 | head -n1 > par1f2.log
```

Comment:
please don't direct stderr and stdout to the same file, or we'd be unable to check what goes where

Reply:
I have updated the names of the files to make it clearer, while standard error is being directed to where standard output is being directed, but then standard output is changed to direct to a stdout file and what was standard error is piped to get only the first line which is redirected to a stderr file.